model/verification/chunkStatus.go

```diff
@@ -8,6 +8,7 @@ import (
 type ChunkStatus struct {
 	ChunkIndex      uint64
 	ExecutionResult *flow.ExecutionResult
+	ChunkLocatorID  flow.Identifier // used to notify back chunk consumer once processing chunk is done.
```

Comment:
ChunkLocatorID can be calculated by ChunkIndex and ExecutionResult, having this as a field seems redundant to me. What about converting it as a method? 

If we are concerned about computing it multiple times, we could use a private field to cache it. 

Also, we'd better reuse the function that calculates the ChunkLocatorID, so that the ID value is consistent

Reply:
Fixed in https://github.com/onflow/flow-go/pull/686/commits/3f8ebe5d94127b40cc3b7c014071e4635e063b91